### PR TITLE
Api

### PR DESCRIPTION
**Pull request by Timowest12**
In this PR I:

 - Created an API endpoint to list all posts for a user.
 - Created an API endpoint to list all comments for a user's post.
 - Create an API endpoint to add a comment to a post. 
 - Added an API token for every user.
 - API responds in JSON

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -12,7 +12,23 @@
...
+        end
+      end
+    end
+  end
+
+  resources :users, only: [] do
+    member do
+      get 'apitoken'
+    end
+  end
```
Comment (Dipeshtwis) [inline config/routes.rb]: - [x] There are so many duplication routes. Please make sure to use restful routes and avoid making different URL for the same route.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Timowest12 & @Kingjosh007, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:+1: The Application is working fine.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (Timowest12) [reply config/routes.rb]: @Dipeshtwis  Removed duplicate routes and cleaned them up to make them more streamlined and efficient

**Review by Dipeshtwis — APPROVED** [review]
Hi @Timowest12 & @Kingjosh007, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**